<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="chkBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>14</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>6</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="greetingRadio">
      <property name="text">
       <string>인사</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="apologizeRadio">
      <property name="text">
       <string>사과</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="thanksRadio">
      <property name="text">
       <string>감사</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="emergencyRadio">
      <property name="text">
       <string>위급</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="weatherRadio">
      <property name="text">
       <string>날씨</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="messageShow">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>70</y>
     <width>601</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
